ec2/spark_ec2.py

```diff
@@ -643,12 +654,14 @@ def setup_cluster(conn, master_nodes, slave_nodes, opts, deploy_ssh_key):
 
     # NOTE: We should clone the repository before running deploy_files to
     # prevent ec2-variables.sh from being overwritten
+    repo_branch="{r} -b {b}".format(r=opts.spark_ec2_git_repo, b=opts.spark_ec2_branch)
+    print "Cloning spark-ec2 scripts from {rb} on master...".format(rb=repo_branch)
     ssh(
         host=master,
         opts=opts,
         command="rm -rf spark-ec2"
         + " && "
-        + "git clone https://github.com/mesos/spark-ec2.git -b {b}".format(b=MESOS_SPARK_EC2_BRANCH)
+        + "git clone {rb}".format(rb=repo_branch)
```

Comment:
I think we need to change this to always clone to `spark-ec2`.

Otherwise, if people fork `spark-ec2` to something with a different name, the cloned repo will have a random name and stuff will break like the `rm -rf` command a few lines up.